packages/web-twig/src/Resources/components/Button/stories/ButtonDisabled.twig

```diff
@@ -1,26 +1,22 @@
-<section class="docs-Section">
-  <h2 class="docs-Heading">Disabled</h2>
-
-  <Grid cols="1" tablet="2" desktop="3">
+<Grid cols="1" desktop="3">
```

Comment:
⚠️ this has to be the same in web and react packages too. But thanks for unifying it with the default story.

Reply:
You are right, I suppose this is going to happen during the subsequent issues.